package.json

```diff
@@ -46,6 +47,7 @@
     "react-native-reanimated": "^2.14.4",
     "react-native-safe-area-context": "^4.4.1",
     "react-native-screens": "^3.18.2",
+    "react-native-select-dropdown": "^3.3.4",
```

Comment:
As far as I can see, this dropdown isn't used. Please remove the dependency if you're not using it.